src/index.html

```diff
@@ -6,9 +6,124 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link 
+      rel="stylesheet" 
+      href="./style.css"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" 
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header
+      class="header">
```

Comment:
don't move attributes on a new line if there is only one attribute
```suggestion
    <header class="header">
```

Reply:
fix it throughout your code